cmd/juju/cloud/add.go

```diff
@@ -297,6 +297,10 @@ func (c *AddCloudCommand) Run(ctxt *cmd.Context) error {
 	}
 
 	if c.controllerName == "" {
+		if !c.Local {
+			ctxt.Stdout.Write(
+				[]byte("There are no controllers running.\nAdding cloud to local cache so you can use it to bootstrap a controller.\n"))
+		}
 		return addLocalCloud(c.cloudMetadataStore, *newCloud)
```

Comment:
Can you also add general confirmation as to what is going on for the normal cases?
Some confirmation type line output with the context Info method (so could be hidden with --quiet).

Cloud 'my-k8s' added to controller 'local'.

Reply:
added for add/update/list